test/unit/counter/ticking.test.js

```diff
@@ -118,4 +85,49 @@ describe('counting', function () {
     expect(component.state().timeDiff).toBe(0)
     expect(component.state().numbers.seconds).toBe(10)
   })
+
+  describe('frozen prop', function () {
+    it('does not count when "frozen" is true', function () {
+      component = mount(<Counter seconds={10} frozen />)
+      jest.runTimersToTime(5000)
+      expect(component.state().timeDiff).toBe(10000)
+      expect(component.state().numbers.seconds).toBe(10)
+    })
+
+    it('stops to count if frozen prop is updated to true', function () {
+      component = mount(<Counter from={0} to={10000} />)
+      component.instance().stop = jest.genMockFunction()
```

Comment:
The easiest way to ensure that all the components are unmounted after each test is to mount components into the same variable every time. Then we can just do

```js
afterEach(function () {
  component.unmount()
})
```